wandb/apis/public/utils.py

```diff
@@ -32,3 +34,20 @@ def parse_s3_url_to_s3_uri(url) -> str:
     s3_uri = f"s3://{bucket_name}/{key}"
 
     return s3_uri
+
+
+def parse_org_from_registry_path(entity: str, project: str, path: str) -> str:
+    """Parse the org from a registry path.
+
+    Args:
+        entity (str): The entity name.
+        project (str): The project name.
+        path (str): The path to parse.
```

Comment:
**(suggestion)** Nice! I think you could mention that the path here could be a project path or an artifact path